test/blackbox-tests/test-cases/melange/merlin.t

```diff
@@ -46,9 +46,7 @@ Paths to Melange stdlib appear in B and S entries without melange.emit stanza
 All 3 modules (Foo, Foo__ and Bar) contain a ppx directive
 
   $ dune ocaml merlin dump-config $PWD | grep -i "ppx"
```

Comment:
Not sure what to do with this test and the one below. They don't make much sense anymore so we can prob remove them?

Reply:
Seems like it. Do we have tests for regular ppx + melange?

Reply:
I cleaned up the test file in https://github.com/ocaml/dune/pull/8992/commits/53c8a93d9c29c1eee2a6d6a31090a38255dde73a.

> Do we have tests for regular ppx + melange?

Yes, see https://github.com/ocaml/dune/blob/53c8a93d9c29c1eee2a6d6a31090a38255dde73a/test/blackbox-tests/test-cases/melange/merlin.t#L105-L165